automated/linux/usb/test-usb-3.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+dd if=/dev/sda1 of=/dev/null status=progress conv=sync iflag=nocache oflag=nocache bs=1k count=1M
+RET=$?
+
```

Comment:
We should check how we can transfer additional data back to the test executor, in this case the average read-speed.